apps/omg_watcher/lib/omg_watcher/db/eth_event.ex

```diff
@@ -44,7 +44,7 @@ defmodule OMG.Watcher.DB.EthEvent do
       join_keys: [root_chain_txhash_event: :root_chain_txhash_event, child_chain_utxohash: :child_chain_utxohash]
     )
 
-    timestamps(type: :utc_datetime)
+    timestamps(type: :utc_datetime_usec)
```

Comment:
What is the reason for this change... last migration script will change it to `timestamptz`
Will the last script work on DB that have `utc_datetime` timestamps?

Reply:
The last migration uses `utc_datetime` which is timezone unaware. Really bad naming on Ecto's part, in my opinion broken. If `utc_datetime`, then what is `naive_datetime`? Anser: also timezone unaware. 😄 

Also we want `utc_datetime_usec` as that is to the microsecond instead of `utc_datetime` which is only to the second.

Reply:
Ah sorry - there is no question - I realized you are tweaking Ecto.Schema not old migrations...
All good!